spec/fixtures/sample.js

```diff
@@ -10,4 +10,20 @@ var quicksort = function () {
   };
 
   return sort(Array.apply(this, arguments));
-};
\ No newline at end of file
+};
+
+foo({
+    atIndentTwo: true
```

Comment:
well, define "correct", because there is absolutely no way to accomplish this indentation if you go with an inductive (line-to-line) approach unless you want to access incorrect indentations like the following:

```js
http.get({
  hostname: 'localhost',
  port: 80,
  path: '/',
  agent: false  // create a new agent just for this one request
}, 
(res) => {
// Do stuff with response
})
```

Here the `(res)` line has to be outdented because a scope ended (induction), but that's not correct. We would want this to still be indented.

Also, I would argue that the content of the object in the first argument should *not* be on the same indentation as the second argument, but rather that all arguments should be on the same indentation (which implies that the content of the first argument should be on indentation two. This becomes more clear when you rewrite this:

```js
http.get(
  {
    hostname: 'localhost',
    port: 80,
    path: '/',
    agent: false  // create a new agent just for this one request
  }, 
  (res) => {
    // Do stuff with response
  }
)
```